gui/elements/_window.py

```diff
@@ -31,9 +32,17 @@ def add_viewer(self):
         """
         viewer = Viewer(self)
         self.viewers.append(viewer)
-        self._qt_central_widget.layout().addLayout(viewer.controls._qt)
-        self._qt_central_widget.layout().addWidget(viewer._qt)
-        self._qt_central_widget.layout().addWidget(viewer.layers._qt)
+
+        # To split vertical sliders, viewer and layerlist, minimumsizes given for demo purposes/NOT FINAL
+        horizontalSplitter = QSplitter(Qt.Horizontal)
+        viewer.controls._qt.setMinimumSize(QSize(60, 60))
+        horizontalSplitter.addWidget(viewer.controls._qt)
+        viewer._qt.setMinimumSize(QSize(500, 500))
+        horizontalSplitter.addWidget(viewer._qt)
+        viewer.layers._qt.setMinimumSize(QSize(150, 150))
```

Comment:
Can we increase the minimum size right here, at least for now, as the layer widget is getting awkwardly cropped on my screen. I have a few fixed width things in there right now which maybe I can get rid of and then everything will look better. I've also been thinking about moving the opacity slider into the "extra" properties which will reduce the width of the layerwidgets too

Reply:
okay, so do you want me to take action on that now? or just left a reminder for future? :D we can set to any value.

Reply:
I'd change it to 250 now if you don't mind. It might change again in the next PR that I just started working on.